tests/unit_tests/data_steward/gcloud/gcs/gcs_test.py

```diff
@@ -0,0 +1,58 @@
+# Python imports
+import mock
+from io import BytesIO
+from unittest import TestCase
+from unittest.mock import patch, MagicMock
+from typing import Callable
+
+# Third party imports
+
+# Project imports
+from gcloud.gcs import StorageClient
+
+
+class GCSTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        # Input parameters expected by the class
+        self.client = StorageClient()
```

Comment:
@MichaelSchmidt82 this is causing your test to fail.  Can you describe what you are wanting to do here so we can figure out the best fix?